File: datagym-frontend/src/app/label-config/model/geometry/LcEntryPoint.ts
import { LcEntryGeometry } from './LcEntryGeometry';
import { LcEntryType } from '../LcEntryType';

export class LcEntryPoint extends LcEntryGeometry {
  static type: LcEntryType = LcEntryType.POINT;

  constructor(lcEntryParentId: string | null, key?: string, value?: string, color?: string, shortcut?: string) {
    super(LcEntryPoint.type, lcEntryParentId, key, value, color, shortcut);
  }
}

File: datagym-backend/src/main/java/ai/datagym/application/labelConfiguration/entity/geometry/LcEntryPoint.java
package ai.datagym.application.labelConfiguration.entity.geometry;

import javax.persistence.DiscriminatorValue;
import javax.persistence.Entity;
import java.io.Serializable;

@Entity
@DiscriminatorValue(value = "point")
public class LcEntryPoint extends LcEntryGeometry implements Serializable {
    public LcEntryPoint() {
        super();
    }
}
